addons/print/wizard/print_order_wizard.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+from openerp import api, fields, models, _
+from openerp.exceptions import Warning
+
+
+class print_order_wizard(models.TransientModel):
```

Comment:
What is the purpose of this one compared to the previous one ? I know you already told me 3 times, but I think some comments / docstrings could help here :p ...